flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/catalog/PostgresCatalog.java

```diff
@@ -258,6 +256,8 @@ private DataType fromJDBCType(ResultSetMetaData metadata, int colIndex) throws S
 		int precision = metadata.getPrecision(colIndex);
 		int scale = metadata.getScale(colIndex);
 
+		// pg types that gets replaced by jdbc driver:
+	    // - decimal => numeric
```

Comment:
The note looks strange after we removed decimal switch, how about remove this note or list all alias :
>    // Postgres jdbc driver maps several alias to real type, we use real type rather than alias.
	// smallint <=> int2
	// integer <=> int4
	// int <=> int4
	// bigint <=> int8
	// float <=> float8
	// boolean <=> bool
	// decimal <=> numeric

Reply:
I wanted to add the other comments in the other PR (pr #11900) to support serial fields..but I can list all of them here and avoid to add some of them in the other PR